recipes/snakeatac_env/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set version = "0.1.1" %}
+{% set sha256 = "7d9c753aceadd0cdd2132f510375e191f1bd9de92e29e43d710433ca770e1f03" %}
+
+package:
+  name: snakeatac_env
+  version: {{ version }}
+
+source:
+  - url: https://github.com/sebastian-gregoricchio/snakeATAC/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+build:
+  noarch: python
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('snakeatac_env', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy >=1.22
+    - mpich
+
+  run:
+    - bcftools
+    - bedtools
+    - bioconductor-copywriter
+    - bioconductor-shortread
+    - bioinfokit
+    - bwa-mem2
+    - bzip2
+    - cutadapt
+    - deeptools >=3.5.2
+    - fastqc
+    - gatk4
+    - macs3
+    - matplotlib-base
+    - multiqc >=1.23
+    - pdfcombine
+    - pip
+    - py-bgzip
+    - py2bit
+    - pybigwig
+    - pyfaidx
+    - pysam
+    - python
+    - r-colorspace
+    - r-data.table
+    - r-dplyr
+    - r-generics
+    - r-ggplot2
+    - r-ggtext
+    - r-plyr
+    - r-rcolorbrewer
+    - r-reshape2
+    - r-scales
+    - r-snow
+    - r-stringr
+    - r-viridis
+    - samtools
+    - scipy
+    - seaborn
+    - snakemake >=7.24.0
+    - snpsift
+    - subread
+    - tobias
+    - ucsc-bedgraphtobigwig
+    - biopython
+    - docutils
+    - fastcluster
+    - logomaker
+    - numpydoc
+    - pandas
+    - pybedtools
+
+
+test:
+  commands:
+    - deeptools --help
+    - samtools --help
+    
+
+about:
+  home: https://github.com/sebastian-gregoricchio/snakeATAC
+  license: https://github.com/sebastian-gregoricchio/snakeATAC/blob/main/LICENSE
```

Comment:
This should be a SPDX id. 
'license_file' should be'LICENSE'

Reply:
Thanks! I updated to:
license:  GPL-3.0-or-later
license_file: LICENSE